packages/app/src/resource/ProfilesPage.tsx

```diff
@@ -0,0 +1,184 @@
+import { OperationOutcome, Resource, ResourceType } from '@medplum/fhirtypes';
+import React, { useCallback, useEffect, useState } from 'react';
+import {
+  Document,
+  SupportedProfileStructureDefinition,
+  ResourceForm,
+  isSupportedProfileStructureDefinition,
+  useMedplum,
+} from '@medplum/react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { showNotification } from '@mantine/notifications';
+import { deepClone, normalizeErrorString, normalizeOperationOutcome } from '@medplum/core';
+import { Button, Code, Group, Stack, Tabs, ThemeIcon } from '@mantine/core';
+import { addProfileToResource, cleanResource } from './utils';
+import { IconCheck } from '@tabler/icons-react';
```

Comment:
Minor nit: run VS Code "organize imports"

(longer conversation about why ESLint doesn't do this / check this)